src/kata2.booleanToWord.js

```diff
@@ -1,3 +1,11 @@
-const booleanToWord = boolean => {};
+const booleanToWord = boolean => {
+    const a = boolean.toString();
+    if (a == 'true')
```

Comment:
This is a good demonstration of using the double equals, rather than triple equals equality check. a is of type boolean and 'true' is of type string, so they are not strictly equals, but Javascript will try to coerce the values to do a comparison here. Although this is the correct way to use a ==. it is not best practice to do this unless absolutely necessary.